third_party/llvm-project/libcxx/include/__availability

```diff
@@ -153,20 +153,23 @@
 
     // This controls the availability of floating-point std::to_chars functions.
     // These overloads were added later than the integer overloads.
-// #   define _LIBCPP_AVAILABILITY_HAS_NO_TO_CHARS_FLOATING_POINT
+    // Set to the same value as chromium upstream repo.
+#   define _LIBCPP_AVAILABILITY_HAS_NO_TO_CHARS_FLOATING_POINT 0
```

Comment:
why do need these changes?

Reply:
These are set to upstream chromium llvm repo (see comment above (line156)

I get a failure:
[elf_loader_sand/1883:0315/184925.186056:FATAL:exported_symbols.cc(491)] Check failed: address. Failed to retrieve the address of '_ZNSt21_LIBCPP_ABI_NAMESPACE22__libcpp_verbose_abortEPKcz'.
 

https://github.com/youtube/cobalt/actions/runs/8300673760/job/22719946704?pr=2484

in evergreen test runs : so seems like this change is needed

Reply:
reopening , this commit is getting us away from chromium upstream. I was looking at the wrong reference files(third_party/libc++ instead of buildtools/third_party/libc++

Reply:
I ended up adding src/verbose_abort to the build source files - that fixed the error